staging/com.smartnews/manifest.tt

```diff
@@ -0,0 +1,37 @@
+class @com.smartnews
+{
+  import loader from @org.thingpedia.v2();
+  import config from @org.thingpedia.config.none();
+
+  list query top_articles(in opt counter: Number #[prompt="How many news articles do you want?"],
+    			  out news_id: String,
+                          out title: String,
+                          out audio: Entity(tt:url),
+                          out date: Date,
+    			  out site_name: String,
```

Comment:
I imagine this will be an entity, so you can track the identifier of the site and the name of the site separately? Perhaps a URL with a display field, or a custom entity?

Reply:
This demo API returns the url of the audio file of this news article and also the url to the original news source website. I guess we don't need the url of the audio file here?

Reply:
I would say you should have the URL to the original website, and use `#_[formatted]` to add a card that links to the article in the web chatbot.
As for the audio URL, I think it depends on what is the intended user experience here for listening to the whole article. Are we treating this like a media player, sort of like listening to a podcast? If so, yeah having the audio URL will be useful, and then we'll figure out how to control playback from the web chatbot and from the smart speaker. There will be some changes required there.